Description: Fix definition of -TEMPLATE_DIRS for Django 1.7
 Adding a comma at the end of the TEMPLATE_DIRS definition makes it work with
 Django 1.7.
Author: Thomas Goirand <zigo@debian.org>
Bug-Debian: http://bugs.debian.org/755651
Forwarded: https://review.openstack.org/#/c/111561/
Last-Update: 2014-08-04

--- horizon-2014.1.1.orig/horizon/test/settings.py
+++ horizon-2014.1.1/horizon/test/settings.py
@@ -88,7 +88,7 @@ STATIC_URL = '/static/'
 MESSAGE_STORAGE = 'django.contrib.messages.storage.cookie.CookieStorage'
 
 ROOT_URLCONF = 'horizon.test.urls'
-TEMPLATE_DIRS = (os.path.join(ROOT_PATH, 'tests', 'templates'))
+TEMPLATE_DIRS = (os.path.join(ROOT_PATH, 'tests', 'templates'),)
 SITE_ID = 1
 SITE_BRANDING = 'Horizon'
 
